GetandTake/appsettings.json

```diff
@@ -6,10 +6,16 @@
     }
   },
   "AllowedHosts": "*",
-  "database": {
+  "Database": {
     "DefaultConnection": "Server=EPTRANKW0038\\SQLEXPRESS;Database=Northwind;Trusted_Connection=True;"
   },
-  "products": {
+  "Products": {
     "maximumAmount": 0
+  },
+  "Host": {
+    "HostUrl": "https://localhost:7005/"
+  },
+  "LogActionFilter": {
```

Comment:
To make it more abstract and reusable in future let's make name of this section like `Logging` or `LoggingParameters`